framework/include/problems/FEProblemBase.h

```diff
@@ -153,6 +153,26 @@ class FEProblemBase : public SubProblem, public Restartable
   FEProblemBase(const InputParameters & parameters);
   virtual ~FEProblemBase();
 
+  enum class KernelCoverageCheckMode
+  {
+    TRUE,
+    ON,
+    FALSE,
+    OFF,
```

Comment:
These four enumerators should be boiled down to two. I would probably pick `TRUE/FALSE` to reflect the user parameter but I don't have a strong preference

Reply:
Please allow me to double-check on that, @lindsayad: With this PR we change the type of two parameters from bool to enum. Moose [input files](https://mooseframework.inl.gov/application_usage/input_syntax.html) allow for bool beside "TRUE" and "FALASE" also "ON" and "OFF". By only picking "TRUE/FALSE", we invalidate a portion of existing Moose input files (the ones using ON/OFF). Also some of the tests currently in use are using "OFF".
Can I remove "ON" and "OFF" and change all tests used by Moose affected?

Reply:
you're right. Let's keep it as is